arwen/contexts/validator.go

```diff
@@ -47,6 +47,22 @@ func (validator *wasmValidator) verifyFunctions(instance wasmer.InstanceHandler)
 	return nil
 }
 
+var protectedFunctions = map[string]bool{
+	"internalVMErrors":  true,
+	"transferValueOnly": true}
```

Comment:
protect the following as well: 	generalSCRIdentifier = "writeLog"
	signalError          = "signalError"
	completedTxEvent     = "completedTxEvent"

Reply:
done